homeassistant/components/switchbot_cloud/sensor.py

```diff
@@ -89,6 +90,13 @@
     native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
 )
 
+POWER_CONSUMPTION_DESCRIPTION_IN_MA = SensorEntityDescription(
```

Comment:
```suggestion
POWER_CONSUMPTION_DESCRIPTION = SensorEntityDescription(
```
It's Milliwatt-Hour, not Milli-Ampere, and since it's the only "POWER_*" description (unlike "CURRENT_*") I think the unit isn't needed for the constants name.

Reply:
great  thank you